packit_service/config.py

```diff
@@ -112,6 +112,9 @@ def hide(token: str) -> str:
             f"server_name='{self.server_name}')"
         )
 
+    def use_stage(self) -> bool:
+        return self.deployment != Deployment.prod
```

Comment:
why not `self.deployment == Deployment.stg`? :)

Reply:
In general: we compare to prod to be sure that we use prod configuration only on prod.
So when you create another 'deployment', like I did when trying the CentOS new OCP4 cluster, Laura's `use_stage()` would return `True`, while yours would return `False` (and prod configuration would be used).

So maybe we need to twist the logic and have `prod` attribute & `use_prod()` instead of `stage` & `use_stage()`?